src/main/java/io/stargate/sgv2/jsonapi/api/model/command/column/definition/ColumnDefinition.java

```diff
@@ -0,0 +1,19 @@
+package io.stargate.sgv2.jsonapi.api.model.command.column.definition;
```

Comment:
Why don't we need the folder definition, if we just have one class ColumnDefinition.java.
Just being curious, like to know how you think this type of questions

Reply:
Thanks, as long as you have a reason for it.